cmake/AutoOptionHelpers.cmake

```diff
@@ -37,18 +37,26 @@ function(_REGISTER_AUTO_OPTION _NAME _FEATURE_VAR _DESCRIPTION _DEFAULT)
   )
 endfunction()
 
-macro(_CHECK_PACKAGE_DEPENDS _OPTION_VAR _PACKAGE_DEPENDS _FEATURE_VAR)
+macro(_CHECK_PACKAGE_DEPENDS _OPTION_VAR _PACKAGE_DEPENDS _FEATURE_VAR _COMPONENT)
```

Comment:
This would be more flexible as a list of components.

Reply:
agreed, but I couldn't get it to work providing two lists